test/nested.html

```diff
@@ -0,0 +1,108 @@
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<html>
+  <head>
+    <title>iron-collapse-nested</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+    <script src="../../web-component-tester/browser.js"></script>
+    <script src="../../test-fixture/test-fixture-mocha.js"></script>
+
+    <link rel="import" href="../../test-fixture/test-fixture.html">
+    <link rel="import" href="../iron-collapse.html">
+  </head>
+  <body>
+
+    <test-fixture id="test">
+      <template>
+        <iron-collapse id="outer-collapse">
+          <div style="height:100px;">
+            Lorem ipsum
+          </div>
+
+          <iron-collapse id="inner-collapse">
+            <div style="height:100px;">
+              consectetur adipiscing elit
+            </div>
+          </iron-collapse>
+        </iron-collapse>
+      </template>
+    </test-fixture>
+
+    <script>
+
+      suite('nested', function() {
+
+        var outerCollapse;
+        var innerCollapse;
+
+        setup(function () {
+          outerCollapse = fixture('test');
+          innerCollapse = outerCollapse.querySelector('#inner-collapse');
+        });
+
+        suite('closed outer collapse', function() {
+
+          test('inner collapse default opened attribute', function() {
+            assert.equal(innerCollapse.opened, false);
+          });
+
+          test('inner collapse default style height', function() {
+            assert.equal(innerCollapse.style.height, '0px');
+          });
+
+          test('set inner opened to true updates inner style height without animation', function() {
+            innerCollapse.opened = true;
+
+            // Animation disabled and height expanded
+            assert.equal(innerCollapse.style.transitionDuration, '0s');
+            assert.equal(innerCollapse.style.height, 'auto');
+          });
+
+        });
+
+        suite('opened outer collapse', function() {
+
+          setup(function () {
+            outerCollapse.opened = true;
+          });
+
+          test('inner collapse default opened attribute', function() {
+            assert.equal(innerCollapse.opened, false);
+          });
+
+          test('inner collapse default style height', function() {
+            assert.equal(innerCollapse.style.height, '0px');
+          });
+
+          test('set inner opened to true triggers height animation', function() {
+            innerCollapse.opened = true;
```

Comment:
Also, can you write this same test for a horizontal innerCollapse & check it has a non-zero width?